srtcore/core.cpp

```diff
@@ -8666,17 +8665,17 @@ void CUDT::addLossRecord(std::vector<int32_t>& lr, int32_t lo, int32_t hi)
 void CUDT::checkACKTimer(uint64_t currtime_tk)
 {
     if (currtime_tk > m_ullNextACKTime_tk  // ACK time has come
-        // OR the number of sent packets since last ACK has reached
-        // the congctl-defined value of ACK Interval
-        // (note that none of the builtin congctls defines ACK Interval)
-        || (m_CongCtl->ACKInterval() > 0 && m_iPktCount >= m_CongCtl->ACKInterval()))
+            // OR the number of sent packets since last ACK has reached
+            // the congctl-defined value of ACK Interval
+            // (note that none of the builtin congctls defines ACK Interval)
+            || (m_CongCtl->ACKIntervalPkts() > 0 && m_iPktCount >= m_CongCtl->ACKIntervalPkts()))
     {
         // ACK timer expired or ACK interval is reached
         sendCtrl(UMSG_ACK);
         CTimer::rdtsc(currtime_tk);
 
-        const int ack_interval_tk = m_CongCtl->ACKPeriod() > 0
-            ? m_CongCtl->ACKPeriod() * m_ullCPUFrequency
+        const int ack_interval_tk = m_CongCtl->ACKIntervalMicrosec() > 0
+            ? m_CongCtl->ACKIntervalMicrosec() * m_ullCPUFrequency
```

Comment:
As you can see here, the named here `ACKIntervalPkts` has a meaning "maximum number of packets to be kept between two ACKs". The named here `ACKIntervalMicrosec` is "time distance to the next ACK" (regardless whether the current ACK was triggered by timeout or exceeded number of packets). I think this difference should be reflected in the names and therefore can't both be named "Interval".

My proposal: `ACKInterval` -> `ACKMaxPackets`; `ACKPeriod` -> `ACKPeriodTimeout` (plus `_us` suffix, if needed).

Reply:
- ACKInterval -> ACKMaxPackets ✔️ 
- ACKPeriod -> ACKPeriodTimeout (plus _us suffix, if needed). - **maybe `ACKTimeout_us()`**?

Reply:
Should work.